ci/eval/default.nix

```
@@ -163,6 +209,8 @@ let
           done
         ) &
 
+        trap "kill %%" EXIT
```

This isn't going to work if anyone adds another background job later on, since the jobspec here is evaluated at EXIT time instead of right now.

```suggestion
        trap "kill $(jobs -p %%)" EXIT
```

Also I just want to point out that stdenv installs a default exit handler that this overwrites. That may not matter at all, if nothing installs a `failureHook` or `exitHook`, but if you want to preserve that you could switch this over to adding both a `failureHook` and `exitHook` (or just a `failureHook` and adding an explicit kill after the `xargs` finishes).